app/trip/[id]/components/trip-details-header.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  Heading,
+  Tag,
+  Box,
+  Flex,
+  useColorModeValue,
+  Image as ChakraImage,
+  Link,
+  IconButton
+} from '@chakra-ui/react'
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+import { FiEdit3, FiShare2, FiTrash2 } from 'react-icons/fi'
+import { MdManageAccounts, MdAccountCircle } from 'react-icons/md'
+import { formatDateToDayMonthYear } from '@/libs/utils'
+
+type TripDetailsHeaderProps = {
+  id: string
+  image: string | null | undefined
+  title: string
+  dateFrom: string | null | undefined
+  dateTo: string | null | undefined
+  users: {
+    id: number | undefined
+    image: string | null | undefined
+  }[]
+
+  tags: { id: number | undefined; name: string | undefined }[]
+}
+
+export const TripDetailsHeader = ({
+  id,
+  image,
+  title,
+  dateFrom,
+  dateTo,
+  tags,
+  users
+}: TripDetailsHeaderProps) => {
+  const router = useRouter()
+
+  const imageSrc = useColorModeValue(
+    '/images/no_image_light.jpg',
+    '/images/no_image_dark.jpg'
+  )
+
+  const color = useColorModeValue('black', 'gray.300')
+  const tagBgColor = useColorModeValue('primary.700', 'primary.800')
+
+  return (
+    <Box>
+      <Box
+        w={{ base: 'auto', md: '100%' }}
+        h={{ base: '164px', md: '288px' }}
+        position="relative"
+        ml={{ base: 'calc(50% - 50vw)', md: '0px' }}
+        mr={{ base: 'calc(50% - 50vw)', md: '0px' }}
+      >
+        <Image
+          src={image || imageSrc}
+          alt={`image of ${title}`}
+          fill
+          style={{ objectFit: 'cover' }}
+        />
+        <Box
+          position={{ base: 'absolute', md: 'relative' }}
+          backgroundColor={'rgba(0, 0, 0, 0.5)'}
+          top={0}
+          left={0}
+          right={0}
+          bottom={0}
+        />
+      </Box>
+      <Box mt={{ base: '-68px', md: '30px' }}>
+        <Flex
+          justifyContent="space-between"
+          alignItems={{ base: 'center', md: 'start' }}
+          position="relative"
+        >
+          <Box>
+            <Heading
+              as="h1"
+              fontSize={{ base: '2xl', md: '4xl' }}
+              color={{ base: 'gray.100', md: color }}
+            >
+              {title}
+            </Heading>
+            <Box
+              mt={{ base: '4px', md: '8px' }}
+              fontSize={{ base: 'sm', md: 'md' }}
+              color={{ base: 'gray.100', md: color }}
+            >
+              {formatDateToDayMonthYear(dateFrom)} -{' '}
+              {formatDateToDayMonthYear(dateTo)}
+            </Box>
+          </Box>
+
+          <Flex gap={{ base: '10px', md: '28px' }}>
```

Comment:
I find those icons a bit too big. In the Figma design, they're 34px, but here they are 48px on desktop. Perhaps it's better to make them a bit smaller.

Reply:
Well I think 38px is too small if you make icon with rounded background.. So I did it 48px on purpose! Should be bigger than tags